frontend/assets/javascripts/src/modules/form.js

```diff
@@ -98,10 +101,15 @@ define([
                 });
 
             }
+
             if (form.hasStripeCheckout) {
+
                 curl('js!stripeCheckout').then(function () {
                     stripe.init();
                 })
+
+                accordion.init();
+
```

Comment:
Clash of styles here 😬, I prefer the whitespace, I think Python is onto a good thing there 🐍.